### PR TITLE
Replace natural beheading with cranial fissures (splitting your skull)

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request

Replaces natural beheading (doing a lot of damage to the head) with cranial fissures, which split your skull in half.

dreamseeker_2023-12-31T19-38-16

While you have this wound, your eyes can be pulled out of your head with bare hands, and slipping will spill your brain out of your head.

Removes beheading objective from traitor.

Zombies can be beheaded all the same.

Any other way of beheading, such as surgery or amputation shears, is still possible.

Closes #80439 

## Why It's Good For The Game

I've enjoyed seeing the results of the test merge to remove natural beheading. At 947 beheadings in a week, 1 in every 7 deaths resulted in a beheading. This makes it significantly easier to remove people from rounds as the brain is generally critical to actually reviving someone. While round removal is fine, it should be something that is intentionally performed with enough effort. Mass round removals ought to require a significant amount of effort. There are plenty of ways to round remove someone, but it ought not be an incidental choice.

The effects of beheading removal have shown some interesting anecdotal highlights (nothing here is backed up with stats, so take it with a grain of salt):
1. As a whole, people did not really replace it with anything. The biggest alternative has been lighting people on fire, but this has not been done at a significant enough scale at all to be a noteworthy problem, and is still fixable with enough effort.
2. Cult and rev rounds have played out far more interestingly. In one round I was adminning a head of staff who intended to behead every revolutionary they saw, but because they didn't, the back and forth continues. I've also had opportunities to revive head revs that I am skeptical would've been available to me otherwise.

Complete removal of beheading was not chosen as it does not feel right for a repeated fire axe to the head to do basically nothing. The current implementation is intended to be something that is just not useful enough to do on everyone you see, but is still appreciably an effect.

## Changelog
:cl:
balance: Instead of too much damage to the head beheading someone, it will now split their skull in half. While their skull is open, you can rip out their eyes with your hands. and they will spill their brain out of their head if they slip.
balance: The Path of Blades ascension will accept either a beheaded person, or someone with their skull split open.
del: Removed the beheading objectives from traitor.
/:cl:

### CODE REVIEW

File: code/_globalvars/traits/_traits.dm
Comment (Mothblocks) [inline code/_globalvars/traits/_traits.dm:111]: alphabetized this while i was here

**Comment by dragomagol** [comment]
How does this affect the Arena shuttle? I know it's rare, but it relies on beheading to even access the shuttle console

**Comment by Mothblocks** [comment]
Oh yeah I forgot about that, I'll poke at it

**Comment by Admiral-Gigaton** [comment]
Funky ass sprite ahh, good shit, but ngl doesn't look like a split head. Is the split going mouth to nape? That could work, also does hea duplicating disable hats and masks, and make them fall off?

Having a brain bucket item as a gimmick thing would be cool. Get head split, revive, get bucket, have a debuff (slight blindness? Or a lower chance of brain spilling when slipping?) Yeah a good ghetto thing would be the ability to put on a bucket to lower chances of brain spillage so you can get to medbay

Then again, your head is split, how the fuck are you alive and walking to get a brain bucket? Still funny idea

**Comment by Xkallubar** [comment]
Test if the Vorpal Scythe death knell works with this

**Comment by 13spacemen** [comment]
I don't really think beheading corresponds to lack of revival/round removal
Very few people will go out of their way to behead and then hide the head in a locker or other place where it'll never be found
People mainly behead because it looks cool and makes you the antag seem more scary

Antags who go on beheading sprees just leave the head lying around the body, if nobody is picking up the head it's usually because:

1. The small head sprite is obscured by many other bodies, or by a potted plant
2. People just don't care to revive

On the LRP servers it's common for people to just ignore your dead body altogether, even non-beheaded ones
So I don't really think beheading corresponds to round removal, murderboners go from kill to kill, they don't have time to behead and then hide your head in a secret locker (and most don't even want to do that)

**Comment by Loafin34** [comment]
i like the cranial fissures idea, especially since you can take people's eyes out really easily which practically RRs people (try fighting someone blind)

**Comment by necromanceranne** [comment]
> Test if the Vorpal Scythe death knell works with this

If amputation shears work, then the scythe works.

**Comment by ATH1909** [comment]
> I don't really think beheading corresponds to lack of revival/round removal Very few people will go out of their way to behead and then hide the head in a locker or other place where it'll never be found People mainly behead because it looks cool and makes you the antag seem more scary
> 
> Antags who go on beheading sprees just leave the head lying around the body, if nobody is picking up the head it's usually because:
> 
> 1. The small head sprite is obscured by many other bodies, or by a potted plant
> 2. People just don't care to revive
> 
> On the LRP servers it's common for people to just ignore your dead body altogether, even non-beheaded ones So I don't really think beheading corresponds to round removal, murderboners go from kill to kill, they don't have time to behead and then hide your head in a secret locker (and most don't even want to do that)

maybe we should make dismembered heads harder to miss, then? can we give them some kind of priority to make them always show up at the top of item stacks?

**Comment by 13spacemen** [comment]
> > I don't really think beheading corresponds to lack of revival/round removal Very few people will go out of their way to behead and then hide the head in a locker or other place where it'll never be found People mainly behead because it looks cool and makes you the antag seem more scary
> > Antags who go on beheading sprees just leave the head lying around the body, if nobody is picking up the head it's usually because:
> > 
> > 1. The small head sprite is obscured by many other bodies, or by a potted plant
> > 2. People just don't care to revive
> > 
> > On the LRP servers it's common for people to just ignore your dead body altogether, even non-beheaded ones So I don't really think beheading corresponds to round removal, murderboners go from kill to kill, they don't have time to behead and then hide your head in a secret locker (and most don't even want to do that)
> 
> maybe we should make dismembered heads harder to miss, then? can we give them some kind of priority to make them always show up at the top of item stacks?

yeah that sounds good, could also have most crew getting some kind of text or alert saying "You feel disturbed and uneasy at the sight of [X's head]"

and roboticists, coroners, doctors could be exempt from that

**Comment by SomeoneYouProbablyKnow** [comment]
> Zombies can be beheaded all the same.

Eh. Do Husked zombies revive? they do right? Consider making it so that they don't. Fighting zombies with fire and burning the dead are staples of zombie media and is more fun than hit head 10 times.

> this will severely affect the head pike/spear econmy

The guillotine will save us. I believe

> Funky ass sprite ahh

It's true. Ayyy looking man

> and slipping will spill your brain out of your head.

Any way to yoink the brain out of a dead person? Mainly for MMIs (when you're not a robot but just a traitor with a dream and a jug of mannitol)

All in all, good. RRing with a shuttle is much more fun than click head till sepearate. More deliberate too. Opens up space for more fun stuff like that in the future.

**Comment by SimplySmiley123** [comment]
> I don't really think beheading corresponds to lack of revival/round removal Very few people will go out of their way to behead and then hide the head in a locker or other place where it'll never be found People mainly behead because it looks cool and makes you the antag seem more scary
> 
> Antags who go on beheading sprees just leave the head lying around the body, if nobody is picking up the head it's usually because:
> 
> 1. The small head sprite is obscured by many other bodies, or by a potted plant
> 2. People just don't care to revive
> 
> On the LRP servers it's common for people to just ignore your dead body altogether, even non-beheaded ones So I don't really think beheading corresponds to round removal, murderboners go from kill to kill, they don't have time to behead and then hide your head in a secret locker (and most don't even want to do that)

Thing is it's TG, players tend to just, not notice heads on the floor and then bring headless corpses to medbay without bringing the head alongside even when it was at the scene.
And yes it was way easier to round remove someone as you did _not_ have to get the brain out, you just had to chop off the head, put it in your inventory, and then do whatever you want with that, dust it, deep fry it, hide it, or do nothing at all and either survive or die and even then not have the person's head reattached anyway.

**Comment by Mothblocks** [comment]
> Eh. Do Husked zombies revive? they do right? Consider making it so that they don't (Maybe lower husk threshold or add a fire damage multiplier). Fighting zombies with fire and burning the dead are staples of zombie media and is more fun than hit head 10 times.

I don't know but if they don't that's out of scope for the PR--I just remember people saying that zombie rounds were a lot harder in the no beheading PR

> Any way to yoink the brain out of a dead person? Mainly for MMIs (when you're not a robot but just a traitor with a dream and a jug of mannitol)

Surgery/amputation shears will be the easiest

**Comment by SomeoneYouProbablyKnow** [comment]
> that's out of scope for the PR

Fair. It just leaves a weird inconsistency. 

> Surgery/amputation shears will be the easiest

Yeah so it's a gear check. Shame. Wish there was a way to keep it easy for MMIs specifically but no way to do that selectively.

**Comment by Mothblocks** [comment]
Arena shuttle chainsaw now directly beheads

**Comment by EricZilla** [comment]
I had a round where a shadow ran around stealing people's heads, then got cremated, RRing 4 extra people because no one bag checked, so I'm very for this.
Having your ability to play puppy guarded sucks

**Comment by itseasytosee** [comment]
The mechanics associated with this wound seem pretty arbitrary and non-obvious.

**Comment by YakumoChen** [comment]
Hats with the SNUG_FIT flag should prevent brain slipping

**Comment by GPeckman** [comment]
How do you actually fix a cranial fracture?

**Review by Wallemations — APPROVED** [review]
the visual is funny, bro got hit by an anvil

**Comment by DaCoolBoss** [comment]
The real issue is beheading weapons being too common. Captain sword is the big offender because it's used almost every round and is so strong in combat that using it repeatedly on someone's head is often the best way to win a fight. And then most players for whatever reason just go for the beheading the corpse if it's not happened already as a killing blow.

**Comment by GuillaumePrata** [comment]
I'm ambivalent on the beheading part, I hate how people overuse it, but it is also something interesting once in a blue moon.

But having your skull cracked open and losing your brain on a slip? Likely my favorite Mothblock PR for that alone, it fits too well with SS13 wackiness while adding more to the sandbox.

We really should have more wounds with small interactions like this, it is such a smart angle to improve wounds.

**Comment by ophaq** [comment]
I feel like the visual should be more similar to the psyker head.

**Comment by Mothblocks** [comment]
> How do you actually fix a cranial fracture?

surgery

File: code/datums/wounds/cranial_fissure.dm
Comment (Time-Green) [inline code/datums/wounds/cranial_fissure.dm:39]: How does this interact with monkeys or spacers? Or hats and glasses? Looks like this doesn't modifiy the head, but the final mob sprite and just assumes the head is always in the same place (it's not ;_;)

You might want to consider applying it to the head bodypart and blocking headgear

Adding your filter to the output of head/get_limb_icon will do this

**Comment by DATA-xPUNGED** [comment]
This is a funny sprite but it doesn't really look like your head was cracked open in two, just that it was squished

**Comment by thElasT13** [comment]
Whats next? Remove antags? or just remove killing? Not sure how beheading is a problem in a bone, usually you dont get revived in a murderbone because med falls apart and not being headless.

**Comment by Mothblocks** [comment]
yeah those two are my next prs

**Comment by RikuTheKiller** [comment]
Meh, too goofy IMO, I don't really like this.
Beheading is better. I don't want to see it replaced nor removed.
Anything but this, what the fuck is this.

File: code/datums/wounds/cranial_fissure.dm
Comment (Mothblocks) [reply code/datums/wounds/cranial_fissure.dm:39]: I tried making it work with the head bodypart but the rendering code very much expects to be collecting into an overlays list at the end. I tried the exact description you were talking about, but it doesn't work because get_list_icon is returning a list for overlays. I tried combining them into one and then putting that on a mutable_appearance, but that also just didn't wrok.

Comment (Time-Green) [reply code/datums/wounds/cranial_fissure.dm:39]: Yeah okay, that's the end of that then. I suppose it's fine for now

**Comment by Overuseddart** [comment]
Very unintuitive, how is a player that doesn’t read this PR and hasn’t played before a potential merge know that zombies specifically can be beheaded, and how would they know what weapons to use? Also ballistic nerf as you can’t behead with higher quality guns such as the .357.

**Comment by Mothblocks** [comment]
they would see it happen once and then probably piece it together from there

**Comment by NoxVS** [comment]
> they would see it happen once and then probably piece it together from there

Schrödinger’s average player is simultaneously incapable of learning new strategies while also being willing to carry around an entire gibber in their pocket to be assembled on the spot

**Comment by itseasytosee** [comment]
You should get someone else to make a sprite.

**Comment by SangryD** [comment]
I have a couple of ideas for the next PRs:
1) Replace all melee with hugs and all projectiles with flying kisses. 
2) Make all LRP servers into MRP so the majority of coders and maintainers, who only plays there, didn't have to balance the game with consideration of pesky LRP players.
3) Rename the /tg/ into Friendship Station, to show that it's not affiliated with 4channel.

**Comment by Mothblocks** [comment]
> > they would see it happen once and then probably piece it together from there
> 
> Schrödinger’s average player is simultaneously incapable of learning new strategies while also being willing to carry around an entire gibber in their pocket to be assembled on the spot

not sure what on earth you're talking about considering i'm replying to the fact that zombies can be beheaded trivially

File: code/datums/wounds/cranial_fissure.dm
Comment (Mothblocks) [reply code/datums/wounds/cranial_fissure.dm:20]:
```suggestion
/// A wound applied when receiving significant enough damage to the head.
/// Will allow other players to take your eyes out of your head, and slipping
/// will cause your brain to fall out of your head.
/datum/wound/cranial_fissure
```

**Comment by breadcrombs** [comment]
might as well remove explosive gibbing and explosive breaches when people inevitably start spamming the most underpowered grenades that can still gib

**Comment by ATH1909** [comment]
> And yes it was way easier to round remove someone as you did _not_ have to get the brain out, you just had to chop off the head, put it in your inventory, and then do whatever you want with that, dust it, deep fry it, hide it, or do nothing at all and either survive or die and even then not have the person's head reattached anyway.

Aren't heads bulky items?

**Comment by ATH1909** [comment]
> Captain sword is the big offender because it's used almost every round and is so strong in combat that using it repeatedly on someone's head is often the best way to win a fight.

im sorry what

what meta have you been living in for the past few years

**Comment by SomeoneYouProbablyKnow** [comment]
> > Captain sword is the big offender because it's used almost every round and is so strong in combat that using it repeatedly on someone's head is often the best way to win a fight.
> 
> im sorry what
> 
> what meta have you been living in for the past few years

I mean that really easy to explain. Man's got a lot of bad experiences with trigger happy capd who chopped his head off, that's it. Which is its own problem to be fair but not to be blamed on sword, a unique item.

The big problem child of beheading is medical saw anyway, mass producable, small, inconspicuous, high wounding, high damage. Sheesh

**Comment by SomeoneYouProbablyKnow** [comment]
The salt prices will never recover after this. You've oversupplied the market

**Comment by Admiral-Gigaton** [comment]
> > And yes it was way easier to round remove someone as you did _not_ have to get the brain out, you just had to chop off the head, put it in your inventory, and then do whatever you want with that, dust it, deep fry it, hide it, or do nothing at all and either survive or die and even then not have the person's head reattached anyway.
> 
> Aren't heads bulky items?
> 
> EDIT:
> 
> > I had a round where a shadow ran around stealing people's heads, then got cremated, RRing 4 extra people because no one bag checked, so I'm very for this.
> 
> Wait, they're not bulky?
> 
> ... Would making them bulky be a potential fix for that, then?

Me tossing someone's head into space or the bottom of an icebox pit

**Comment by Xkallubar** [comment]
My salt business is going bankrupt, everyone already has a lifetime supply of salt now!

**Comment by Dawnseer** [comment]
Shear stocks are going up!

**Comment by EuSouAFazer** [comment]
First cloning now beheading? What's next, removing Cult?

**Comment by SomeoneYouProbablyKnow** [comment]
> First cloning now beheading? What's next, removing Cult?

I WISH

**Comment by gorillion** [comment]
> First cloning now beheading? What's next, removing Cult?



> First cloning now beheading? What's next, removing Cult?

IZW_8Zm-7ixy6f6GqQd0tL5ZHYYPPVUsbwjE-3Bb9Fw

**Comment by CCC23** [comment]
you forgot to change the eyes for the ashen eyes

**Comment by DeCEll-1** [comment]
ATH1909s and 13spacemens suggestions can literally fix all the problem there is.

& if users dont get the head to the medbay for borging or something, making brain spill out harder doesnt make them bring head more likely.

yes they will survive more since they can run to the medbay, but how many times someone survived from an attack that could split someones head in half?

Also in my personal experience i dont even remember getting beheaded.

**Comment by LePiromano** [comment]
this is fucking hilarious but it shouldn't completely replace beheading imo

**Pull request merged**
